build.sbt

```diff
@@ -40,7 +35,6 @@ val commonSettings = Seq(
 import com.typesafe.sbt.packager.archetypes.systemloader.ServerLoader.Systemd
 val buildDebSettings = Seq(
   Debian / serverLoading := Some(Systemd),
-  debianPackageDependencies := Seq("openjdk-8-jre-headless"),
```

Comment:
not sure we need to depend on anything, if it's not there it just errors out (which it would anyway if there was no JRE)
I guess it provides some protection from using the wrong one, but I decided to remove anyway.